src/app/modules/main/chat_section/module.nim

```diff
@@ -724,12 +728,7 @@ method onNewMessagesReceived*(self: Module, sectionIdMsgBelongsTo: string, chatI
     return
 
   # Badge notification
-  let messageBelongsToActiveSection = sectionIdMsgBelongsTo == self.controller.getMySectionId() and 
-    self.controller.getMySectionId() == self.delegate.getActiveSectionId()
-  let messageBelongsToActiveChat = self.controller.getActiveChatId() == chatIdMsgBelongsTo
-  if(not messageBelongsToActiveSection or not messageBelongsToActiveChat):
-    let hasUnreadMessages = unviewedMessagesCount > 0
-    self.updateBadgeNotifications(chatIdMsgBelongsTo, hasUnreadMessages, unviewedMentionsCount)
+  self.updateBadgeNotifications(chatIdMsgBelongsTo, unviewedMessagesCount > 0, unviewedMentionsCount)
```

Comment:
You'll have conflicts here with my commits. Basically, we check the `muted` value for the Message count, but not the mentions counts.